cmd/release-notes/README.md

```diff
@@ -89,6 +89,7 @@ level=debug timestamp=2019-07-30T04:02:44.3716249Z caller=notes.go:497 msg="Excl
 | output                  | OUTPUT          |                     | No       | The path where the release notes will be written                                                                                  |
 | format                  | FORMAT          | go-template:default | Yes      | The format for notes output (options: json, go-template:inline:<template-string> go-template:path/to/template.file)               |
 | release-version         | RELEASE_VERSION |                     | No       | The release version to tag the notes with                                                                                         |
+| dependencies            |                 | false               | No       | Add dependency report                                                                                                             |
```

Comment:
This flag's default is documented as false, but my experience suggests it's actually true (as does the code below)

Reply:
Thank you for the hint, @grantr. I proposed a fix in https://github.com/kubernetes/release/pull/1305